instance/models/openedx_appserver.py

```diff
@@ -295,40 +294,317 @@ def get_playbooks(self):
             playbooks.append(self.lms_user_creation_playbook())
         return playbooks
 
+    def _get_configuration_variables(self):
```

Comment:
You could consider moving this particular function to a separate module again.  This file is already rather hard to navigate, and this is the biggest settings block of all.

Reply:
I've moved this into a mixin which is imported and subclassed by OpenEdxAppServer. Is this a suitable loading strategy?